docs/designs/link-output.md

```diff
@@ -42,11 +42,14 @@ Docfx generate `links` information of each files into one single file `.links.js
 
 *link model*:
 
-  |                         |                      |
-  |-------------------------|----------------------|
-  | `source_url`            | /dotnet/articles/a   |
-  | `source_moniker_group`  | 218c13d0             |
-  | `target_url`            | /dotnet/articles/b   |
+  |                         |                                         |
+  |-------------------------|----------------------                   |
+  | `source_url`            | /dotnet/articles/a                      |
+  | `source_moniker_group`  | 218c13d0                                |
+  | `target_url`            | /dotnet/articles/b                      |
+  | `source_file_url`       | https://github.com/dotnet/articles/a.md |
+  | `source_line`           | 10                                      |
```

Comment:
`source_line` will be null if not applicable

Reply:
Planning to set 1 if null

Reply:
that's okay to me - but would be better we call it out as well as the index is one-based

Reply:
the `source_line` is from `SourceInfo`, the comments says it is [one based start](https://github.com/dotnet/docfx/blob/805251fa713f08ac58894762c6673c64713268b7/src/docfx/lib/log/SourceInfo.cs#L18)